src/api/v1/themes.go

```diff
@@ -0,0 +1,106 @@
+package v1
+
+import (
+	"github.com/labstack/echo"
+	"net/http"
+)
+
+type (
+	theme struct {
+		Background  background `json:"background,omitempty"`
```

Comment:
Thoughts on making this a `[]background` slice? A client could maybe implement it as a slideshow or something.

Reply:
IMO fanciness like slideshows makes more sense as a "cycle between these themes" rather than "cycle through the background, text color, etc of this one theme"

Reply:
or maybe background itself could look more like 

```go
background {
    Speed uint
    Images []string
    Colors []uint32
}
```

or maybe theme could have a type, and `group`/`slide` types could parent/child each other?

```go
theme{
    Type: GROUP,
    Delay: 40,
    Slides: []theme{
        {
//            Type: SLIDE  //DEFAULT
            Background: {
                URL: "hello/world.png",
                Color: 0x011001,
            },
        },
    },
}```

Reply:
IMO this is overengineered /shrug

Reply:
> IMO fanciness like slideshows makes more sense as a "cycle between these themes" rather than "cycle through the background, text color, etc of this one theme"

I agree, ideally you could select which themes get cycled through

Reply:
Sounds good